server/app/views/AlertComponent.java

```diff
@@ -0,0 +1,123 @@
+package views;
+
+import static j2html.TagCreator.div;
+import static j2html.TagCreator.h2;
+import static j2html.TagCreator.h3;
+import static j2html.TagCreator.h4;
+import static j2html.TagCreator.h5;
+import static j2html.TagCreator.h6;
+import static j2html.TagCreator.p;
+
+import com.google.common.collect.Lists;
+import j2html.tags.specialized.DivTag;
+import java.util.Optional;
+import services.AlertType;
+import views.style.BaseStyles;
+
+/**
+ * Render a USWDS Alert Component
+ *
+ * @see <a
+ *     href="https://designsystem.digital.gov/components/alert">https://designsystem.digital.gov/components/alert</a>
+ */
+public final class AlertComponent {
```

Comment:
Thought: I wonder if a builder would be nice for this component... it would make the calling code a lot clearer (replacing the /* varname */ comments) and make it easy to leave out the optional fields. But maybe overkill.